pkg/datasource/datasource_test.go

```diff
@@ -398,7 +398,7 @@ func TestDatasource(t *testing.T) {
 	})
 
 	t.Run("getTrace query", func(t *testing.T) {
-		response, err := queryDatasource(ds, datasource.QueryGetTrace, datasource.GetTraceQueryData{Query: "traceID"})
+		response, err := queryDatasource(ds, datasource.QueryGetTrace, datasource.GetTraceQueryData{Query: "trace1"})
```

Comment:
Had to fix this test as I had to remove hardcoded `traceId` in line [185](https://github.com/grafana/x-ray-datasource/pull/332/files#diff-61e64a895c97b57c9a8193f4fdd8bf4fc2d8ae4a5c82d27e85650d0c38c272deL185)